backend/src/server/server.ts

```diff
@@ -0,0 +1,10 @@
+import http from 'http';
```

Comment:
I downloaded the repo in order to test it with my frontend solution and I got these errors when running `npm run dev`:
![image](https://user-images.githubusercontent.com/28508893/177784685-524a5db5-9240-40df-9396-6a0b7b8aeda8.png)